src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
+        runDuke();
+    }
+
+    public static void printWelcome() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        printDividerLine();
+        System.out.println("Hello! I'm Duke! \n " + logo);
+        System.out.println("What can I do for you?");
+        printDividerLine();
+
+    }
+
+    public static void printDividerLine() {
+        System.out.println("___________________________________ \n");
+    }
+
+    public static void printGoodbye() {
+        printDividerLine();
+        System.out.println("Bye. Hope to see you again soon! :)");
+        printDividerLine();
+    }
+
+    public static ArrayList<Task> loadDukeMemory() {
+        printDividerLine();
+        ArrayList<Task> taskList = FileHelper.retrieveData();
```

Comment:
You can consider using plural for the name of that variable since it is an ArrayList of many things